examples/src/main/java/tachyon/examples/BasicNonByteBufferOperations.java

```diff
@@ -93,33 +89,25 @@ private void write(TachyonFileSystem tachyonClient) throws IOException, TachyonE
     }
   }
 
-  private FileOutStream getOrCreate(TachyonFileSystem tachyonFileSystem, TachyonURI filePath,
-      boolean deleteIfExists, OutStreamOptions clientOptions) throws IOException, TachyonException {
-    TachyonFile file;
-
-    try {
-      file = tachyonFileSystem.open(filePath);
-    } catch (Exception e) {
-      file = null;
-    }
-    if (file == null) {
+  private FileOutStream getOrCreate(FileSystem fileSystem, TachyonURI filePath,
```

Comment:
Why this is called `getOrCreate`? At first sight, I thought this means "if a file exits, get outstream for that file, otherwise, create a file at path, and get its outstream."